### PR TITLE
Add News & Events to UniversalContentItem

### PR DESCRIPTION
**Pull request by lmommaerts**
**DO NOT MERGE**

[Issue](https://3.basecamp.com/3926363/buckets/22579289/todos/4433063022)

Adds the News & Events callout to UniversalContentItem slug pages.

See /fall-offering-and-the-legacy-park-project for an example.

### CODE REVIEW

**Comment by redreceipt** [comment]
@lmommaerts do you want me to review this? It says "do not merge" at the top

**Comment by lmommaerts** [comment]
@redreceipt Sorry, yeah, or anyone else on the team you think might be good to review it.

Our process moving forward will be:
- Review PR
- Notify LH we're ready to deploy
- Upon given OK to deploy, merge PR
- Notify LH fix is live

So, once the PR is approved, I'll notify Chris, who will notify LH. Then, once we get approval, I'll merge the PR and let Chris known.

**Review by redreceipt — APPROVED** [review]

**Pull request merged**
